pkgs/applications/science/electronics/openroad/default.nix

```diff
@@ -79,7 +80,12 @@ mkDerivation rec {
   ];
 
   patches = [
-    ./0001-Fix-string-formatting-in-tests.patch
+    (fetchpatch {
+      name = "openroad-fix-fmt-10.patch";
+      url = "https://github.com/The-OpenROAD-Project/OpenROAD/commit/9396f07f28e0260cd64acfc51909f6566b70e682.patch";
```

Comment:
Have you opened a PR for this patch? Couldn't find a link... Would be nice to have it in a comment.

Reply:
Added a link now. The patch works but the maintainers would like a different approach.
I would like to work on that independently of this PR.